src/platform/forms/save-in-progress/SaveInProgressDevModal.jsx

```diff
@@ -116,13 +118,13 @@ const SipsDevModal = props => {
               value={textData}
               onInput={e => handlers.onChange(e.target.value)}
             />
-            <Select
+            <VaSelect
               label="Return url"
               name="sips_url"
               options={availablePaths}
               value={{ value: sipsUrl }}
               includeBlankOption={false}
-              onValueChange={value => setSipsUrl(value.value)}
+              onVaSelect={value => setSipsUrl(value.value)}
```

Comment:
Does the returning value actually work? The structure in a web component is `event.detail.value` for a custom event https://design.va.gov/about/developers/using-web-components#custom-events